### PR TITLE
Add a supporters section to the frontpage

### PR DESCRIPTION
**Pull request by jaimergp**
We'll need this for the upcoming Zulip transition (because we are a sponsored OSS project). Adapted from the conda-forge website; it's nice to be using the same framework so this collaboration is seamless :)

Includes logos from companies sponsoring developing time in the @conda/steering-council and others sponsoring infrastructure.

### CODE REVIEW

**Review by beckermr — APPROVED** [review]

**Review by xhochy — APPROVED** [review]

**Review by beeankha — APPROVED** [review]

File: src/components/Supporters/index.jsx [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+    dark: "img/supporters/quantstack_dark.svg",
+    width: 250,
+  },
+  {
+    name: "Prefix.dev",
+    link: "https://prefix.dev/",
+    light: "img/supporters/prefix_light.svg",
+    dark: "img/supporters/prefix_dark.svg",
+    width: 250,
+  },
```
Comment (jaimergp) [inline src/components/Supporters/index.jsx:83]: @wolfv is it okay to also mention y'all here?

File: src/components/Supporters/index.jsx [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+    dark: "img/supporters/quansightlabs_dark.svg",
+    width: 250,
+  },
+  {
+    name: "QuantStack",
+    link: "https://quantstack.net/",
+    light: "img/supporters/quantstack_light.svg",
+    dark: "img/supporters/quantstack_dark.svg",
+    width: 250,
+  },
```
Comment (jaimergp) [inline src/components/Supporters/index.jsx:76]: @Hind-M, is it okay to also mention y'all here?

File: src/components/Supporters/index.jsx [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+  //   dark: "img/supporters/ioos.png",
+  //   width: 150,
+  // },
+  {
+    name: "NVIDIA",
+    link: "https://www.nvidia.com/",
+    light: "img/supporters/nvidia_light.svg",
+    dark: "img/supporters/nvidia_dark.svg",
+    width: 250,
+  },
```
Comment (jaimergp) [inline src/components/Supporters/index.jsx:55]: @jakirkham is it okay to also mention y'all here?

File: src/components/Supporters/index.jsx [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+    dark: "img/supporters/anaconda_dark.svg",
+    width: 250,
+  },
+  // {
+  //   name: "IOOS Integrated Ocean Observing System",
+  //   link: "https://ioos.noaa.gov/",
+  //   light: "img/supporters/ioos.png",
+  //   dark: "img/supporters/ioos.png",
+  //   width: 150,
+  // },
```
Comment (jaimergp) [inline src/components/Supporters/index.jsx:48]: @ocefpaf is it okay to also mention IOOS here?
```suggestion
  {
    name: "IOOS Integrated Ocean Observing System",
    link: "https://ioos.noaa.gov/",
    light: "img/supporters/ioos.png",
    dark: "img/supporters/ioos.png",
    width: 150,
  },
```

File: src/components/Supporters/index.jsx [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+    dark: "img/supporters/nvidia_dark.svg",
+    width: 250,
+  },
+  // {
+  //   name: "Voltron Data",
+  //   link: "https://voltrondata.com/",
+  //   light: "img/supporters/voltron_light.svg",
+  //   dark: "img/supporters/voltron_dark.svg",
+  //   width: 250,
+  // },
```
Comment (jaimergp) [inline src/components/Supporters/index.jsx:62]: @mariusvniekerk is it okay to also mention y'all here? If that's the case, could you also add that info to [`steering.csv`](https://github.com/conda/governance/blob/7ee27ebfecc3b36f52d926ee4d36eec30e457439/steering.csv#L4)? Thanks!

**Comment by wolfv** [comment]
Thanks @jaimergp - excellent stuff!

**Review by Hind-M — APPROVED** [review]

File: src/components/Supporters/index.jsx (hunk 2 above)
Comment (Hind-M) [reply src/components/Supporters/index.jsx:76]: Yep! Thanks!

**Review by trallard — APPROVED** [review]

**Pull request merged**